pallets/reward-distribution/src/lib.rs

```diff
@@ -70,7 +112,25 @@ pub mod pallet {
 	}
 
 	#[pallet::error]
-	pub enum Error<T> {}
+	pub enum Error<T> {
+		//Overflow
+		Overflow,
+		//underflow
+		Underflow,
+		//if origin tries to withdraw with 0 rewards
+		NoRewardsForAccount,
+		//error if amount to be minted is more than rewarded in total
+		NotEnoughRewardsRegistered,
+	}
+
+	#[pallet::hooks]
+	impl<T: Config> Hooks<T::BlockNumber> for Pallet<T> {
+		fn on_initialize(n: T::BlockNumber) -> Weight {
+			Self::execute_on_init(n);
+			//TODO benchmark this weight properly
```

Comment:
You already added some benchmark. Are you not happy with it or is this a stale comment @gianfra-t?

Reply:
It is a leftover comment. The only "concern" is that the generated file outputted a [strange](https://github.com/pendulum-chain/spacewalk/pull/415/files#diff-d012ec42d9339a0c96c67bf1afef24aed64425b409839559c22b7cf9545417e5R118)  set of commented lines which I have not seen before. Otherwise it should be okay.